documentation/src/docs/prepare-development-environment.md

```diff
@@ -9,10 +9,12 @@ you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
     http://www.apache.org/licenses/LICENSE-2.0
 -->
+import { IfHaveFeature, IfMissingFeature, SDPVersion } from 'nautilus-docs';
 
 import PrepareDevelopmentEnvironmentPrefix from '../snippets/spark-connectors/prepare-development-environment-prefix.md';
-
+<IfHaveFeature feature="nautilus">
```

Comment:
There's a better way of accomplishing this. Simply create a blank file https://github.com/pravega/docs/tree/master/docs-website/docs/snippets/spark-connectors/prepare-development-environment-prefix.md.

Reply:
IfHaveFeature reverted, blank snippet provided here: https://github.com/pravega/docs/pull/1/commits/9abd9cb452a18c1e0a074355a4aaa59c37a3a8c6#diff-d2466ecd9d8fa63bbc4e351dd1aa7abdbbe72f78e4f31aaa36e85b2a87174ec7